src/ensembl/src/content/app/entity-viewer/gene-view/GeneView.tsx

```diff
@@ -233,10 +229,6 @@ const GeneViewWithData = (props: GeneViewWithDataProps) => {
       return;
     }
 
-    if (isSidebarOpen) {
-      dispatch(closeSidebarModal());
-    }
```

Comment:
Couldn't find anywhere in the ticket or in the description of the PR the reason for this change. Don't we want to close the modal for EntityViewer when we switch from it to a different application anymore?

Reply:
As discussed during daily scrum today we are keeping the modal open